example/callback.c

```diff
@@ -78,7 +78,10 @@ int main(int argc, const char* argv[]) {
   fseek(file, 0L, SEEK_SET);
   wasm_byte_vec_t binary;
   wasm_byte_vec_new_uninitialized(&binary, file_size);
-  fread(binary.data, file_size, 1, file);
+  if(fread(binary.data, file_size, 1, file) != 1) {
```

Comment:
Nit: space after `if` (here and elsewhere).

Reply:
Done!